.circleci/test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
can you put in a comment a high-level description of what's going on :)?

Reply:
@rdzhabarov Sure, I will do that once I finished editing code.